### PR TITLE
[bug 1134475] Cleanup all references to input-dev environment

### PR DESCRIPTION
**Pull request by lgp171188**
Remove all references to input-dev.allizom.org

### CODE REVIEW

File: fjord/base/templates/contribute.json [hunk 1]
```diff
@@ -23,7 +23,6 @@
     "urls": {
         "prod": "https://input.mozilla.org/",
         "stage": "https://input.allizom.org/",
```
Comment (willkg) [inline fjord/base/templates/contribute.json]: JSON is finicky and doesn't like trailing commas. So we need to also nix the comma on line 25.

**Comment by lgp171188** [comment]
Oops, I didn't notice that. Fixed now

**Comment by willkg** [comment]
Awesome! Thank you!

**Comment by willkg** [comment]
c1fe25a [bug 1134475] Cleanup all references to input-dev environment

**Pull request merged**